component-models.json

```diff
@@ -712,16 +712,327 @@
         "component": "text",
         "valueType": "string",
         "name": "jcr:title",
-        "label": "Page Title",
+        "label": "Title",
+        "required": true,
         "description": "Sets the title of the page in page meta."
       },
+      {
+        "component": "text",
+        "valueType": "string",
+        "name": "jcr:pagetitle",
```

Comment:
"jcr:pageTitle" was throwing lint error hence needed to change the name to "jcr:pagetitle"